test/e2e/framework/framework.go

```diff
@@ -32,34 +32,41 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	clientcmd "k8s.io/client-go/tools/clientcmd"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	"k8s.io/kubernetes/pkg/cloudprovider"
+
+	oci "github.com/oracle/oci-cloud-controller-manager/pkg/oci" // register oci cloud provider
```

Comment:
Qualifying the import here is redundant.

Reply:
Not entirely; it was origionally imported as `_` to register the cloud provider (via `init`). It's now imported as `oci` to call `oci.ProviderName()` but I feel the qualification is worthwhile because the main reason for importing the package is the side effect of the import.